docs/user-guide/advanced-guide/2fa-auth.rst

```diff
@@ -0,0 +1,85 @@
+Two-factor authentication
+=========================
+
+If you prefer to :doc:`/user-guide/cloud` then server security should be improved. You should add two-factor authentication
+to your server. If you use ``Amazon Web Services`` as a cloud service, follow |google_authenticator_aws| instead of the following section.
+
+.. |google_authenticator_aws| raw:: html
+
+   <a href="https://aws.amazon.com/blogs/startups/securing-ssh-to-amazon-ec2-linux-hosts/" target="_blank">this guide</a>
+
+First of all, install ``Google Authenticator`` (|google_authenticator_app_android|, |google_authenticator_app_ios|) on your mobile phone.
+Open the application, click on ``Begin setup`` and be ready to scan the ``QR-code`` with the ``Scan barcode`` button.
+
+.. |google_authenticator_app_android| raw:: html
+
+   <a href="https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2&hl=en" target="_blank">Android</a>
+
+.. |google_authenticator_app_ios| raw:: html
+
+   <a href="https://itunes.apple.com/ru/app/google-authenticator/id388497605?mt=8" target="_blank">iOS</a>
+
+Log in to the server as you previously did during installation of the node:
+
+.. code-block:: console
+
+   $ ssh root@95.179.156.74
```

Comment:
@eugene-babichenko, all guides have real `IP-address` and `IP-address` should be presented because on the screenshots also `IP-address` on droplets/instance/servers (Digital Ocean, AWS, Vultr). Users will be confused if on the screen it would be `IP` but in the command example `example.com`.